### DISCUSSION TITLE
默认启用生电所需配置 | Technical needs config is enabled by default

### DISCUSSION THREAD

**Discussion by MC-XiaoHei** [LeavesMC/Lumina#18] [Polls]
Chinese ver.
鉴于 Lumina 主要方向为生电，我提议默认启用某些配置以实现默认即接近原版。例如 [paper-unsupported-settings](https://docs.papermc.io/paper/reference/global-configuration#unsupported_settings) 中的大多数。

这是一个表决，您可以选择“同意”或“不同意”（我们希望听到您不同意的意见）

English ver.
Given that Lumina's primary orientation is technical, I propose to enable certain configurations by default to achieve a default approach to the vanilla. For example, most of [paper-unsupported-settings](https://docs.papermc.io/paper/reference/global-configuration#unsupported_settings).

This is a vote, you can either "Agree" or "Disagree" (we want to hear if you disagree)

**Comment by LittleChest** [comment]
**与Leaves同步**

**Comment by MC-XiaoHei** [comment]
请注意：这个功能与Leaves的生电模式并不完全相同，我们只是修改相关配置默认值，而不是完全覆盖
Please note: This feature is not exactly the same as Leaves' mc-technical-mode, we are only changing the relevant configuration defaults, not overriding them completely

**Comment by LittleChest** [comment]
- 不修改默认值（启动时生成的配置与Folia一致）
- 添加生电配置 开启后无论配置项是什么都将覆盖Lumina推荐的配置 此配置默认开启

但是你不会支持这个的😅

**Reply by MC-XiaoHei** [reply]
> * 不修改默认值（启动时生成的配置与Folia一致）
> * 添加生电配置 开启后无论配置项是什么都将覆盖Lumina推荐的配置 此配置默认开启
> 
> 但是你不会支持这个的😅

我不认为强制覆盖是好的 这导致意义不明

**Comment by MC-XiaoHei** [comment]
表决结束，议题通过。
